CODE_OF_CONDUCT.md

```diff
@@ -35,19 +42,21 @@ The conda Organization is dedicated to providing a harassment-free community for
 
 Thank you for helping make this a welcoming, friendly community for all.
 
-**CODE OF CONDUCT REPORTING FORM:** [*https://URL/to/your/CoC/reporting/form*](https://URL/to/your/CoC/reporting/form)
+## [Report a code of conduct incident here.](https://form.jotform.com/221527028480048)
```

Comment:
Just wondering who maintains this form, in case we want to improve it in the future?

> "You can enter Anonymous if you wish to report something anonymously. We will keep your personal information confidential even if you provide your contact information."

Can we make the name form input optional instead of having to write "Anonymous" into it?

> "This is optional, but can be very helpful for followup communication. If you are reporting anonymously, then please consider creating and using a random/new email account here."

I'm not sure if this is reasonable, the form should allow submitting anonymously without having to create a new email account.

Reply:
Alternatively an alias like conduct@conda.org that sends the reports to the CoC Committee members may be useful to set up as well.

Reply:
@jezdez, some responses:

> Just wondering who maintains this form, in case we want to improve it in the future?

Right now it is owned by condaconduct@gmail.com which is the shared email address for the committee.  That will be set up to distribute any reports to all current committee members.  We were thinking of having the co-chairs have access to it, with a third member being the password recovery option for it.

> Alternatively an alias like [conduct@conda.org](mailto:conduct@conda.org) that sends the reports to the CoC Committee members may be useful to set up as well.

I think that long term a conda.org email would be better, but right now I want to get this out the door.  Also see my general response in this PR.

> Can we make the name form input optional instead of having to write "Anonymous" into it?

Done.

> I'm not sure if this is reasonable, the form should allow submitting anonymously without having to create a new email account.

The form does not require an email address.  The form says this is optional and then asks them to *consider* creating a new email account.  So, I'm not sure what the objection is?

Reply:
> @jezdez, some responses:
> 
> > Just wondering who maintains this form, in case we want to improve it in the future?
> 
> Right now it is owned by [condaconduct@gmail.com](mailto:condaconduct@gmail.com) which is the shared email address for the committee. That will be set up to distribute any reports to all current committee members. We were thinking of having the co-chairs have access to it, with a third member being the password recovery option for it.
> 
> > Alternatively an alias like [conduct@conda.org](mailto:conduct@conda.org) that sends the reports to the CoC Committee members may be useful to set up as well.
> 
> I think that long term a conda.org email would be better, but right now I want to get this out the door. Also see my general response in this PR.

Fully agreed, and that makes sense. Thank you!

> > Can we make the name form input optional instead of having to write "Anonymous" into it?
> 
> Done.
> 
> > I'm not sure if this is reasonable, the form should allow submitting anonymously without having to create a new email account.
> 
> The form does not require an email address. The form says this is optional and then asks them to _consider_ creating a new email account. So, I'm not sure what the objection is?

I'm mostly trying to put myself in the shoes of the reporting party and wonder if the mere suggestion to create an email account could feel like a barrier for reporting, that's all really. It's not a big worry though.